tasks/version_shell.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
```

Comment:
Shebang does not require bash here, so open to interpretation (default `/bin/sh` is `dash` on Ubuntu), @bastelfreak .